modules/tinymce/src/themes/silver/test/ts/headless/components/iframe/IframeTest.ts

```diff
@@ -64,7 +76,8 @@ describe('headless.tinymce.themes.silver.components.iframe.IFrameTest', () => {
           s.element('iframe', {
             classes: [
               arr.has('tox-dialog__iframe'),
-              transparent ? arr.not('tox-dialog__iframe--opaque') : arr.has('tox-dialog__iframe--opaque')
+              transparent ? arr.not(`${baseClassPrefix}opaque`) : arr.has(`${baseClassPrefix}opaque`),
```

Comment:
As much as I hate it, this is probably better than writing the string twice
```suggestion
              (transparent ? arr.not : arr.has)(`${baseClassPrefix}opaque`),
```

Reply:
Ah this an interesting way to refactor the code, thanks for the suggestion.